Microsoft.Toolkit.Uwp.UI.Controls.DataGrid/DataGrid/DataGrid.cs

```diff
@@ -862,6 +865,28 @@ private static void OnColumnWidthPropertyChanged(DependencyObject d, DependencyP
             dataGrid.EnsureHorizontalLayout();
         }
 
+        /// <summary>
+        /// Gets or sets the amount of data to fetch for virtualizing/prefetch operations.
+        /// </summary>
+        /// <returns>
+        /// The amount of data to fetch per interval, in pages.
+        /// </returns>
+        public double DataFetchSize
+        {
+            get { return (double)GetValue(DataFetchSizeProperty); }
+            set { SetValue(DataFetchSizeProperty, value); }
+        }
+
+        /// <summary>
+        /// Identifies the <see cref="DataFetchSize"/> dependency property
+        /// </summary>
+        public static readonly DependencyProperty DataFetchSizeProperty =
+            DependencyProperty.Register(
+                "DataFetchSize",
+                typeof(double),
+                typeof(DataGrid),
+                new PropertyMetadata(DATAGRID_defaultDataFetchSize));
```

Comment:
There must be an OnDataFetchSizePropertyChanged method like OnMaxColumnWidthPropertyChanged: throw when e.NewValue is double.NaN or is < 0, and restore e.OldValue the same way.

Reply:
Done